docs/aerodromes/classc/Amberley.md

```diff
@@ -26,44 +26,87 @@ AMB ADC owns the Class C airspace **in the AMB CTR** within 10nm of the YAMB ARP
 Aircraft planned via **BN**, **JEDDA**, **MESED**, **BOBOP**, **TATEN**, shall be assigned the **Procedural SID** that terminates at the appropriate waypoint.
 Aircraft who are not planned via those points or who are negative RNAV may be assigned a RADAR SID or a visual departure. 
 
-#### Coded Departures (Fast Jets)
-Visual Departures are commonly requested by high performance military jets and often processed in the form of a coded departure (e.g. BYRON7 departure), which can be found in the AD2 Sups Page for YAMB. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east of Brisbane, avoiding civilian traffic inbound to Brisbane. 
-
-Aircraft will depart the circuit visually and track overhead YAMB to begin tracking for their initial waypoint.
+### Coded Departures (Fast Jets)
+Visual Departures are commonly requested by high performance military jets and often processed in the form of a coded departure (e.g. BYRON1 departure), which can be found in the AD2 Sups Page for YAMB. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east of Brisbane, avoiding civilian traffic inbound to Brisbane. 
 
 !!! phraseology
-    CRNG21 plans to enter R637 via the gate MOSSI for Military Training.
-    **AMB ADC**: "CRNG21, cleared to YAMB via MOSSI, flight planned route. Runway 15, Northern 1 Departure. Climb to `F190`. Squawk 3601, Departure Frequency 126.20"  
+    CRNG21 plans to enter R637 via the gate MOSSI for Military Training.  
+    **AMB ADC**: "CRNG21, cleared YAMB via MOSSI, flight planned route. Runway 15, Northern 3 Departure. Climb `F190`. Squawk 3601, Departure Frequency 126.20"   
+
+Aircraft will depart the circuit visually and track overhead YAMB to begin tracking for their initial waypoint.
 
 !!! phraseology
-    WOLF03 was assiged the BYRON7 departure with their initial clearance. 
+    WOLF03 was assiged the BYRON1 departure with their initial clearance.  
     **AMB ADC**: "WOLF03, make left turn, reach `F190` by COWIE, cleared for takeoff"  
     **WOLF03**: "Make left turn, reach `F190` by COWIE, cleared for takeoff, WOLF03"  
```

Comment:
@llamavert 
Just doing a final review now. Is this constraint phraseology required? It seems the coded clearance includes the constraint, which would suggest there's no need to reiterate it with a takeoff clearance

![image](https://github.com/user-attachments/assets/f28d3caa-f4b1-449d-b90c-c29b8c274d1d)

Reply:
The height requirement was always in the phraseology examples before i made the pull request. I do agree with you however in a VATSIM environment it still may be good to reiterate the height requirement for pilots who may be unaware or have just forgot.

Reply:
Yeah ok, no worries. I might remove it for consistency with other positions.